### DISCUSSION TITLE
How to modify the proportion of the training set in the training process (write in input.json)

### DISCUSSION THREAD

**Discussion by foreverfreedown** [deepmodeling/deepmd-kit#596] [Q&A]
I have one or two training sets that have less data and I want to increase their weight. How should I set that in the input file?Thanks a lot.

**Comment by marian-code** [comment]
As far as I know there is no way to do this in current stable release. A simple workaround would be to just include the structures you want to put emphasis on multiple times. It's not a very nice solution but it will get the job done, at least to some degree, without making changes to the code.

**Reply by foreverfreedown** [reply]
Thank you for your answer, Can I copy this training set into multiple training sets to join the training?

**Reply by marian-code** [reply]
Yes, that should work.

**Reply by foreverfreedown** [reply]
Thank you for your reply. I will try.
